Kustomize does not get built in MakeFile

/kind bug

**What steps did you take and what happened:**
When I try to run `make release-manifests` I expect the `kustomze` binary to be downloaded. But the command fails with 

> hack/tools/bin/kustomize build config > out/infrastructure-components.yaml
bash: hack/tools/bin/kustomize: No such file or directory
Makefile:315: recipe for target 'release-manifests' failed
make: *** [release-manifests] Error 127


**What did you expect to happen:**
The command should build the binary and execute the command. Requires just a minor fix to `MakeFile` which does not invoke $KUSTOMIZE.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/Makefile
+++ b/Makefile
@@ -269,7 +269,7 @@ release: clean-release  ## Builds and push container images using the latest git
 	$(MAKE) release-manifests
 
 .PHONY: release-manifests
-release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
+release-manifests: $(RELEASE_DIR) $(KUSTOMIZE) ## Builds the manifests to publish with a release
 	$(KUSTOMIZE) build config > $(RELEASE_DIR)/infrastructure-components.yaml
 
 .PHONY: release-staging

